qdrant-landing/content/documentation/guides/capacity-planning.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Capacity Planning
+weight: 11
+aliases:
+  - capacity
+  - cloud/capacity-sizing
```

Comment:
The URL currently is https://qdrant.tech/documentation/cloud/capacity-sizing/
```suggestion
  - /documentation/cloud/capacity-sizing
```

Reply:
https://deploy-preview-1195--condescending-goldwasser-91acf0.netlify.app/documentation/cloud/capacity-sizing is broken